cmd/server-main.go

```diff
@@ -319,6 +318,14 @@ func initAllSubsystems(ctx context.Context, newObject ObjectLayer) (err error) {
 		return fmt.Errorf("Unable to list buckets to heal: %w", err)
 	}
 
+	if globalIsDistErasure {
+		for _, bucket := range buckets {
```

Comment:
Maybe let's add some information message, when it starts and when it ends, the user will wait when he has a lot number of buckets.